pkg/skaffold/constants/constants.go

```diff
@@ -73,6 +73,9 @@ const (
 	GithubIssueLink = "https://github.com/GoogleContainerTools/skaffold/issues/new"
 
 	Windows = "windows"
+
+	// HaTS is the HaTS Survey ID
+	HaTS = "hats"
```

Comment:
I think we should just move this into `survey/config.go` alongside the `hatsURL`.

Reply:
done